security/botan/Portfile

```diff
@@ -68,6 +68,28 @@ configure.args-append --docdir=share/doc \
                       --with-sqlite3 \
                       --with-zlib
 
+# List of all intrisics that can be disabled
+set intrinsics {}
+lappend intrinsics sse2
+lappend intrinsics ssse3
+lappend intrinsics sse4.1
+lappend intrinsics sse4.2
+lappend intrinsics avx2
+lappend intrinsics bmi2
+lappend intrinsics rdrand
+lappend intrinsics rdseed
+lappend intrinsics aes-ni
+lappend intrinsics sha-ni
+lappend intrinsics altivec
+lappend intrinsics neon
+lappend intrinsics armv8crypto
+lappend intrinsics powercrypto
+
```

Comment:
the change looks good - but, can this not just be `set intrinsics { item1 item2 item3 .. itemn }` instead of using `lappend`; that would increase the readability?

Reply:
Definitely a good idea for a future update. But didn't want to hold up the update, given that the overall approach is sound.